python/flatnessTask.py

```diff
@@ -34,9 +34,13 @@ def flatnessTask(sensor_id, infile, dtype='OGP', pickle_file=None):
     #
     azims = (10, 45)
     for azim in azims:
-        sensorData.flatness_plot(azim=azim)
+        sensorData.flatness_plot(azim=azim, title='Surface Plot, %s' % sensor_id)
         metData.plot.save('%s_flatness_point_cloud_azim_%i.png'
                           % (sensor_id, azim))
 
     if pickle_file is not None:
         sensorData.persist(pickle_file)
+
```

Comment:
I've removed it, likewise for the example code in flatnessTask_delta.py and qaPlot.py.  They are not useful for testing the code beyond demonstrating that it runs.